### PR TITLE
Add instructions reset of compose state

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: localDev/compose/README.md [hunk 1]
```diff
@@ -8,6 +8,12 @@
...
 - Run docker ps
-- Run docker stop {SERVICE_NAME}
+- Run docker stop {SERVICE_NAME} 
+  - (The services that are currently avaliable for swapping can be found in utils/servicePortMapping.sh
 - Run ./utils/runProxy.sh
 - Run service with correctly configured ip addresses and port mappings
+
+## Stop developing a service (Return to original state) without reset
+- Run docker ps -a (The proxy service will be called {SERVICE_NAME}_PROXY)
+- Run docker stop {{SERVICE_NAME}_NAME}
```
Comment (pallenSL) [inline localDev/compose/README.md]: Should this be "{{SERVICE_NAME}_PROXY}"

Comment (pallenSL) [inline localDev/compose/README.md]: Would {SERVICE_NAME} and {PROXY_NAME} be simpler?

**Comment by jebbinSCL** [comment]
Adjusted

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**